modules/src/wiki.py

```diff
@@ -1,14 +1,20 @@
 import wikipedia
+import string
 from templates.text import TextTemplate
 
 def process(input, entities):
     output = {}
     try:
         query = entities['wikipedia_search_query'][0]['value']
-        data = wikipedia.summary(query)
+        data = wikipedia.page(query)
+        template = TextTemplate()
+        printable = set(string.printable)
+        s = filter(lambda x : x in printable, data.summary)
```

Comment:
<img width="1270" alt="screen shot 2016-05-03 at 11 18 11 am" src="https://cloud.githubusercontent.com/assets/4915950/14993664/ca0bde64-1120-11e6-9e5d-34314d431b98.png">

This is what happens when we don't remove non-printable characters and don't get the ["text"] attribute on get_message